app/controllers/courses_controller.rb

```diff
@@ -0,0 +1,6 @@
+class CoursesController < ApplicationController
```

Comment:
What is the use of this controler

Reply:
For the index page of the app which request the user for signup/login